documentation/feature/laserpointer.md

```diff
@@ -6,21 +6,18 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### 1.1 Visible light laser
+ACE3 adds visible light laser, what's even better is that BI's laser pointers and supported modded ones also get this feature.
```

Comment:
Confusing sentence..
Maybe try
"ACE3 adds visible light laser. This feature is compatible with BI's lasers as well as supported modded ones." or something similar.

p.s. this feature is kinda broken (#bisfault) so it might never work fully...